src/lib/rust.ts

```diff
@@ -285,9 +285,17 @@ const tryDownloadingBinary = async ({
 
   const url = `https://github.com/fluencelabs/marine/releases/download/${name}-v${version}/${name}-${platformToUse}-x86_64`;
 
+  startSpinner(
+    `Trying to download pre-built ${name}@${version} binary to ${replaceHomeDir(
```

Comment:
```suggestion
    `Downloading pre-built ${name}@${version} binary to ${replaceHomeDir(
```

Reply:
I think "prebuilt" is kinda confusing here: user doesn't need to think about how our CI works